### DISCUSSION TITLE
Contextual help or description.

### DISCUSSION THREAD

**Discussion by mrg9999** [ChrisTitusTech/winutil#212] [Ideas]
I was thinking it would be nice to have a pop-up/floating window that describes the progams for those not familiar.
e.g. I dind't know what rust or obsidian  were

**Comment by Lakshaya-wq** [comment]
While the idea of a help box might be useful enough, but then you have the well-documented README already. Also, rust is a programming language, and obsidian a knowledge-base (both are programs in short), and documenting each program is out of scope. If you are running this tweak, you probably are aware of the basic windows-things like updates, etc..

**Reply by mrg9999** [reply]
I've added a help button that displays a list of the apps and a brief description as a single page. Since no-one else has requested it, I won't create yet another fork.
